tests/test_rpc.py

```
@@ -0,0 +1,62 @@
+    
+    server = JSONRPCServer(chain, mempool, network)
+    port = free_tcp_port
+    await server.start(host="127.0.0.1", port=port)
+    
+    yield server, port, chain, mempool
+    
+    await server.app.cleanup()
+
+@pytest.mark.anyio
+async def test_rpc_blockNumber(rpc_server):
+    server, port, chain, mempool = rpc_server
```

Ruff (0.15.15)

[warning] Unpacked variable `server` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)